tests/dummy/app/templates/docs/scroll-into-view.md

```diff
@@ -19,6 +19,29 @@ You should use this modifier whenever you need to have an element scrolled into
 `shouldScroll` can be either a Boolean or a Promise that resolves to a truthy or falsy value. It does not handle a rejected Promise.
 
 
+## Testing
+`scroll-into-view-mock` provides a function that will mock the native browser `scrollIntoView` and allow testing which elements invoked the modifier
+
+`mockScrollIntoView()` - will mock the native API and return an object with the following 2 functions
+* `scrollIntoViewCalledWith(Element|DOM selector)` - tests if the modifier was invoked on the element
+* `resetMock()` - returns the native scrollIntoView function
+
+```javascript
+import mockScrollIntoView from 'ember-scroll-modifiers/test-support/scroll-into-view-mock';
+
+...
+const mockHelperFunctions = mockScrollIntoView();
```

Comment:
Let's go ahead and use a `beforeEach`/`afterEach` pattern in this example, since that's how we expect people to use it most of the time. They can choose to do it differently, but let's show them the ideal scenario in our docs.